tools/interop_matrix/README.md

```diff
@@ -5,6 +5,21 @@ This directory contains scripts that facilitate building and running gRPC tests
 The setup builds gRPC docker images for each language/runtime and upload it to Google Container Registry (GCR). These images, encapsulating gRPC stack
 from specific releases/tag, are used to test version compatiblity between gRPC release versions.
 
+## Step-by-step instructions for adding a new release to compatibility test
+We have continuous nightly test setup to test gRPC backward compatibility between old clients and latest server.  When a Google developer creates a new gRPC release, s/he is also responsible to add the just-released gRPC client to the nightly test.  The steps are:
```

Comment:
I think it's probably better to not emphasize on Google developer. Can it be changed to "When someone creates a new gRPC release".

Reply:
Understand.  GCR access needs permission, so it is technically googler-only.  Tuned down this part regardless.